README.md

```diff
@@ -81,9 +81,20 @@ export default {
 };
 ```
 
+`AWS_SECRETS_MANAGER_NAME` can also be a list of strings that each correspond to a separate AWS Secret Manager bucket name. Each bucket from the list is evaluated in order. In the scenario that a specific secret appears in two buckets the value will be overwritten.
```

Comment:
I'd add some details on the order the secrets are merged in and which value will "win" when overwritten.